harness/propertyHelper.js

```diff
@@ -47,7 +47,7 @@ function verifyProperty(obj, name, desc, options) {
   var failures = [];
 
   if (Object.prototype.hasOwnProperty.call(desc, 'value')) {
-    if (desc.value !== originalDesc.value) {
+    if (!assert._isSameValue(desc.value, originalDesc.value)) {
```

Comment:
this one depends on `assert` having a public property `_isSameValue` which not documented and might break if any runner uses a custom harness.

I'd be fine to actually expand this code here to verify for `NaN` cases, but still avoiding an abstraction that is not in this file.

Reply:
4f4d2d6